tests/Radebatz/ACache/Tests/Decorators/Psr/PoolIntegrationTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Radebatz\ACache\Tests\Decorators;
+
+use Cache\IntegrationTests\CachePoolTest;
+use Radebatz\ACache\ArrayCache;
+use Radebatz\ACache\Decorators\Psr\CacheItemPool;
+
+class PoolIntegrationTest extends CachePoolTest
+{
+    public function createCachePool()
+    {
+        return new CacheItemPool(new ArrayCache());
+    }
+}
```

Comment:
Happy with all changes but whhere is this class coming from:
````
use Cache\IntegrationTests\CachePoolTest;
```
Any chance to use just ````\PHPUnit_Framework_TestCase````?